src/Homework_5/MyNUnit/TestInfo.cs

```diff
@@ -0,0 +1,15 @@
+namespace MyNUnit;
+
+/// <summary>
+/// Class with information about runned test.
+/// </summary>
+public class TestResult
+{
+    public string? TestName { get; set; }
```

Comment:
```suggestion
    public required string TestName { get; init; }
```

А может быть тест без имени? Если нет, то используй `required`